zomes/chat/src/pagination_helper.rs

```diff
@@ -0,0 +1,113 @@
+//! TODO: Document how to use this crate in general
+//!
+//!
+//!
+use crate::error::ChatResult;
+use chrono::{DateTime, Datelike, Duration, NaiveDateTime, Timelike, Utc};
+use hdk::prelude::*;
+use std::ops::Sub;
+
+///
+pub fn get_page_links(
+    base: Path,
+    earlier_than: Timestamp,
+    target_count: usize,
+) -> ChatResult<Vec<Link>> {
+    let mut links: Vec<Link> = Vec::new();
+    let mut next_path = timestamp_into_path(base.clone(), earlier_than, None)?;
+
+    loop {
+        debug!("Next path: {:?}", next_path.clone());
+        // Ensure the path exists
```

Comment:
```suggestion
        // Confirm the path exists
```
I don't want to use the word "ensure" in the comment because in the path system that term is used to create the path links and entry, which is exactly what we are trying to avoid here!